client/src/App.js

```diff
@@ -1,5 +1,161 @@
+import { veggies } from "./fixtures/AllVeggies.js";
+import { fruities } from "./fixtures/AllFruities.js";
+import { salads } from "./fixtures/AllSalads.js";
+import months from "./fixtures/months.js";
+import PictureCard from "./components/Card.js";
+
+import { useState } from "react";
+
+import styled from "styled-components";
+//import Button from "Button.js";
+
 function App() {
-  return <div className="App"></div>;
+  console.log(veggies, 1);
+  console.log(fruities, 2);
+  console.log(salads, 3);
+
+  // UseState für Veggies - month - fruties - salads
+  const [veggiesToRender, setVeggiesToRender] = useState(veggies);
+  const [frutiesToRender, setFruitiesToRender] = useState(fruities);
```

Comment:
```suggestion
  const [fruitiesToRender, setFruitiesToRender] = useState(fruities);
```
I'm not sure which one would be correct, but you sometimes yous `fruties` and sometimes `fruities`. I would probably stick to one of the two